DeComp/compress.py

```diff
@@ -417,9 +417,10 @@ def _sqfs(self, infodict):
                 self.extension(cmdinfo["mode"])
 
         sqfs_opts = self._sub_other_options(cmdlist.args, cmdinfo)
-        if not infodict['arch']:
-            sqfs_opts.remove("-Xbcj")
-            sqfs_opts.remove("%(arch)s")
+        if infodict['mode'] == "squashfs_xz":
```

Comment:
Rick, why did you limit the next block to only to squashfs_xz?

Reply:
Only the squashfs_xz definition contains "-Xbcj %(arch)s"

https://github.com/dol-sen/pyDeComp/blob/0e536909d829e8c7e96f72e1ff95d1570b1de2ec/DeComp/definitions.py#L199-L230

For other compression types, raises:
```
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/DeComp/compress.py", line 135, in _compress
    return self._run(infodict)
  File "/usr/lib/python3.9/site-packages/DeComp/compress.py", line 202, in _run
    success = func(infodict)
  File "/usr/lib/python3.9/site-packages/DeComp/compress.py", line 421, in _sqfs
    sqfs_opts.remove("-Xbcj")
ValueError: list.remove(x): x not in list
```